prov/efa/src/efa_hmem.c

```diff
@@ -86,6 +99,18 @@ static int efa_hmem_support_status_update_cuda(struct efa_hmem_support_status *c
 	}
 
 	cuda_status->p2p_supported = true;
+	/* Eager and runting read protocols */
+	if (-ENODATA != fi_param_get(&rxr_prov, "inter_max_medium_message_size", &tmp_value)) {
+		EFA_WARN(FI_LOG_DOMAIN,
+		         "The environment variable FI_EFA_INTER_MAX_MEDIUM_MESSAGE_SIZE was set, "
+		         "but EFA HMEM via Cuda API only supports eager and runting read protocols. Please "
+		         "unset the variable and run again.\n");
+		abort();
+	}
+
+	cuda_status->max_medium_msg_size = 0;
+	cuda_status->min_read_msg_size = max_eager_msg_size_with_largest_header(efa_domain) + 1;
```

Comment:
user should be able to modify override this min read msg size via env. Same thing for neuron

Reply:
done